maint-lib/makelib.mk

```diff
@@ -5,21 +5,24 @@ HOST_ARCH ?= $(shell uname --machine)
 
 
 # Export all relevant environment variables from a flavor spec in the format:
-#   HOST_ARCH,CEPH_VERSION,BASEOS_REPO,BASEOS_TAG
+#   HOST_ARCH,CEPH_VERSION_SPEC,BASEOS_REPO,BASEOS_TAG
+# Note that CEPH_VERSION_SPEC is split into CEPH_VERSION and CEPH_POINT_RELEASE
 # Note that this format is the same as the FLAVORS spec format with HOST_ARCH prepended; it's
 #   necessary to pass HOST_ARCH in so we can do parallel cross-builds of different arches later.
 comma := ,
 define set_env_vars
 $(shell bash -c 'set -eu ; \
 	function set_var () { export $$1="$$2" ; echo -n $$1="\"$$2\" " ; } ; \
 	set_var HOST_ARCH         "$(word 1, $(subst $(comma), ,$(1)))" ; \
-	set_var CEPH_VERSION      "$(word 2, $(subst $(comma), , $(1)))" ; \
+	CEPH_VERSION_SPEC="$(word 2, $(subst $(comma), ,$(1)))" ; \
```

Comment:
Following the pattern established below, lower case `ceph_version_spec` here.

Reply:
Readability and maintainability: If upper-case is used here, it may imply to the reader/maintainer that this value is supposed to become an environment variable rather than just being a local value.

Reply:
ok